study.md

```diff
@@ -50,7 +51,8 @@ Does `proxima_centauri.print_mass` run? Why or why not? If so, then what is the
 output? If not, then how can it be fixed?
 
 ```md
-<!-- your answer here -->
+It does not, because mass_kg cannot be accessed outside of the object. To fix it,
```

Comment:
This doesn't run because `mass_kg` is a local variable that doesn't exist in `#print_mass`'s scope.